sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/ChangeFeedBatch.scala

```diff
@@ -64,6 +85,15 @@ private class ChangeFeedBatch
         container
       )
 
+      if(hasBatchCheckpointLocation) {
+        val latestOffsetLocation = Paths.get(changeFeedConfig.batchCheckpointLocation.get, "latestOffset").toString
```

Comment:
for personal understanding:
Do we only write to latestOffset file? 
where is other places we are going to read/write latestOffset file?